panel/src/components/Forms/Input/TagsInput.vue

```diff
@@ -280,7 +284,10 @@ export default {
         this.navigate("last");
         this.$refs.autocomplete.close();
         e.preventDefault();
-        e.target.blur();
+
+        if (this.tags.length !== 0) {
+          e.target.blur();
+        }
```

Comment:
Solution by @distantnative to prevent losing focus

Reply:
I'm wondering if we need this at all... `this.navigate("last")` should already focus the last existing tag... and therefore the input should be blurred already anyways. Maybe we could remove that line completely? We would need to test it.